libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -60,12 +60,16 @@ configuration options. For example:
 --------------------
 monitoring:
   enabled: true
+  cluster_uuid: PRODUCTION_ES_CLUSTER_UUID <1>
   elasticsearch:
-    hosts: ["https://example.com:9200", "https://example2.com:9200"] <1>
+    hosts: ["https://example.com:9200", "https://example2.com:9200"] <2>
     username: {beat_monitoring_user}
     password: somepassword
 --------------------
-<1> This setting identifies the hosts and port numbers of {es} nodes
+<1> This setting identifies the Elasticsearch cluster under which the
```

Comment:
Link to https://www.elastic.co/guide/en/elasticsearch/reference/7.4/cluster-stats.html perhaps? It would be nice to give a user a quick reference to how to get this UUID.

Reply:
Agreed but I'm not sure linking to the Cluster Stats API is the best since it also returns a lot of other information. The simplest API a user could call would be `GET /` but it doesn't look we document that one. So I'm thinking we just mention the API call here itself. WDYT?

Reply:
Yes, I think just mentioning it would be fine. Thanks!

Reply:
Added in 53ef7d4. Let me know what you think.